pkgs/development/python-modules/gunicorn/default.nix

```diff
@@ -8,11 +8,12 @@
 
 buildPythonPackage rec {
   pname = "gunicorn";
-  version = "19.9.0";
+  version = "20.0.4";
```

Comment:
FYI this change breaks apache-airflow build:
```
ERROR: Could not find a version that satisfies the requirement gunicorn<20.0,>=19.5.0 (from apache-airflow==1.10.5) (from versions: none)
ERROR: No matching distribution found for gunicorn<20.0,>=19.5.0 (from apache-airflow==1.10.5)
```

Reply:
opening a ticket is more action-able :), I unfortunately don't have time to look into this :(